python/plugins/processing/algs/gdal/warp.py

```diff
@@ -177,7 +177,7 @@ def getConsoleCommands(self):
             arguments.extend(rastext)
 
         if GdalUtils.version() >= 2000000:
-            if rastext and rastext_crs is not None:
+            if rastext and len(rastext_crs) > 0:
```

Comment:
I wanted to keep it consistent with previous checks of ParameterCrs in the same file ([here](https://github.com/radosuav/QGIS/blob/d9e11d909d043123d7f0e74225663b95c016a2ba/python/plugins/processing/algs/gdal/warp.py#L144) and [here](https://github.com/radosuav/QGIS/blob/d9e11d909d043123d7f0e74225663b95c016a2ba/python/plugins/processing/algs/gdal/warp.py#L147)), but indeed it might be better to update the code at all three spots. But should I then also do the same for other parameter types (e.g. [here](https://github.com/radosuav/QGIS/blob/d9e11d909d043123d7f0e74225663b95c016a2ba/python/plugins/processing/algs/gdal/warp.py#L150))?